SKYKitChat/Classes/UI/ConversationList/SKYChatConversationTableViewCell.swift

```diff
@@ -96,9 +96,9 @@ open class SKYChatConversationTableViewCell: UITableViewCell {
             let participantCount = conversation.participantIds.count
             let info: String
             if participantCount == 1 {
-                info = "1 participant"
+                info = NSLocalizedString("1 participant", comment: "")
             } else {
-                info = "\(participantCount) participants"
+                info = NSLocalizedString("\(participantCount) participants", comment: "")
```

Comment:
One really sad thing about localisation is how bad it cooperate with interpolation. This code will search with strings like "2 participants", "3 participants" in localisation file, it's not ok since we are not going to list all the numbers there.

I would suggest making it `String(format: NSLocalizedString("%d participants", comment: ""), count)`.

Actually it still has problem with plural noun, the ultimate solution would be [this](http://macoscope.com/blog/effective-localization-when-working-with-language-plural-rules/). But given Oursky seldom localises anything, and English is the only language (that we are able to localise) to have rule for plural, I think String(format:) + NSLocalizedString should be good enough.